test/integration/project_index_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class ProjectIndexTest < ActionDispatch::IntegrationTest
+	include Capybara::DSL
+	include FactoryGirl::Syntax::Methods
+
+	test "pagination renders on page" do
+		15.times do 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.